screens/Menu/Settings.js

```diff
@@ -0,0 +1,152 @@
+import React from "react";
+import { View, Text, StyleSheet, Switch } from "react-native";
+import { Button } from "native-base";
+import { Feather } from "@expo/vector-icons";
+import { ScrollView } from "react-native-gesture-handler";
+
+/**
+ * Description: This method holds the toggle switches 
+ * that will compose the setting screen. The screen will have 
+ * also have accessibility to a Google Account Login screen.
+ */
+
+function Settings(props) {
+
+    const [switchVal1, setSwitchVal1] = React.useState(false);
+    const [switchVal2, setSwitchVal2] = React.useState(false);
+
+    const switchLabel1 = switchVal1 ? "ON" : "OFF";
+    const switchLabel2 = switchVal2 ? "ON" : "OFF";
```

Comment:
I think these should be a _var_ since they can change.

Reply:
Good catch!